app/gui/src/project-view/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -439,7 +439,7 @@ declare module '@/providers/widgetRegistry' {
         <div
           ref="activityElement"
           class="activityElement widgetOutOfLayout floatingElement"
-          :style="activityStyles"
+          :style="{ ...activityStyles, '--z-index-selection-submenu': -5 }"
```

Comment:
Z-indexes tend to be a source of bugs. I think it would help to use a named constant here and document the reason for this number.